src/components/slider/slider.tsx

```diff
@@ -574,13 +584,13 @@ export class Slider implements LabelableComponent, FormComponent, InteractiveCom
         <div class="handle-extension" />
         <div class="handle" />
         <span aria-hidden="true" class="handle__label handle__label--minValue">
-          {this.minValue && this.minValue.toLocaleString()}
+          {this.minValue && this.determineGroupSeparator(this.minValue)}
         </span>
         <span aria-hidden="true" class="handle__label handle__label--minValue static">
-          {this.minValue && this.minValue.toLocaleString()}
+          {this.minValue && this.determineGroupSeparator(this.minValue)}
         </span>
         <span aria-hidden="true" class="handle__label handle__label--minValue transformed">
-          {this.minValue && this.minValue.toLocaleString()}
+          {this.minValue && this.determineGroupSeparator(this.minValue)}
```

Comment:
Instead of all the `this.minValue && ` maybe `determineGroupSeparater` could handle when the value is not set?

Reply:
I agree with the suggestion. Looks like it was guarding against it to avoid calling a number on `null`/`undefined`.